<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Joshua's Infection Simulation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pandemic Simulation </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="6" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Simulation Virus Setup</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="sB_init_infected_population">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Simulation Population Setup</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Number of infected people in given population at the beginning of the simulation.</string>
          </property>
          <property name="text">
           <string>Initialy infected people (1-100)</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Virus name</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QSpinBox" name="sB_virus_contagiousness_radius">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="8" column="4">
         <widget class="QPushButton" name="pB_new_virus">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Add New Virus</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="pB_init_sim">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Run Simulation</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="sB_simulation_population">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QTextEdit" name="tE_virus_name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>This parameter describes the distance around a person (+- x and y distance square) in which someone can get infected.</string>
          </property>
          <property name="text">
           <string>Contagiousneess square size (1-100)</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QSpinBox" name="sB_virus_contagiousness">
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="1">
         <widget class="QSpinBox" name="sB_virus_death_rate">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select a Virus</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QPushButton" name="pB_select_virus">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select Virus</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QSpinBox" name="sB_virus_recovery_time">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="9" column="4">
         <widget class="QPushButton" name="pB_update_virus">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Update virus</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Virus death rate (0 - 1000 equivalent to 0.1% to 100%)</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>This parameter describes how long it takes for a person to recover an thus not be contagious anymore.</string>
          </property>
          <property name="text">
           <string>Virus recovery time [Days] (1-1000)</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>This parameter describes the rate at which someone, who comes into contact with an infected person also gets infected. 0.. no one gets infected. 1000.. everyone gets infected</string>
          </property>
          <property name="text">
           <string>Contagiousness (0 - 1000 equivalent to 0.0% to 100%)</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Simulation population size. Dont go over 5000! It will break your computer....</string>
          </property>
          <property name="text">
           <string>Simulation Population (100-10000)</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="cB_virus_selection"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>This parameter is multiplied with a random angle alpha to set a new position (x, y) every day. Every person will move this distance in a random direction.</string>
          </property>
          <property name="text">
           <string>Population mobility (0-250)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="sB_init_population_moving_distance">
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>25</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>sB_simulation_population</tabstop>
  <tabstop>sB_init_infected_population</tabstop>
  <tabstop>cB_virus_selection</tabstop>
  <tabstop>tE_virus_name</tabstop>
  <tabstop>sB_virus_death_rate</tabstop>
  <tabstop>sB_virus_recovery_time</tabstop>
  <tabstop>sB_virus_contagiousness</tabstop>
  <tabstop>sB_virus_contagiousness_radius</tabstop>
  <tabstop>pB_init_sim</tabstop>
  <tabstop>pB_select_virus</tabstop>
  <tabstop>pB_new_virus</tabstop>
  <tabstop>pB_update_virus</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
